diffusion/train.py

```diff
@@ -16,6 +17,37 @@
 from composer.loggers import LoggerDestination
 from composer.utils import dist, reproducibility
 from omegaconf import DictConfig, OmegaConf
+from torch.optim import Optimizer
+
+from diffusion.models.autoencoder import ComposerAutoEncoder, ComposerDiffusersAutoEncoder
+
+
+def make_autoencoder_optimizer(config: DictConfig, model: ComposerModel) -> Optimizer:
+    """Configures the optimizer for use with an autoencoder + discriminator loss."""
+    print('Configuring opimizer for autoencoder+discriminator')
+    assert isinstance(model, ComposerAutoEncoder) or isinstance(model, ComposerDiffusersAutoEncoder)
+
+    # Configure optimizer settings for the autoencoder
+    if hasattr(config, 'autoencoder_optimizer'):
+        autoencoder_param_dict = {k: v for k, v in config.autoencoder_optimizer.items()}
```

Comment:
```suggestion
        autoencoder_param_dict = dict(config.autoencoder_optimizer.items())
```

Reply:
Fixed this, along with the other similar ones. Thanks!